popupbridge-paypaldatacollector-ios.md

```diff
@@ -0,0 +1,48 @@
+## One time payment
+
+Implement the method that receives the popup URL and use this sample code to parse the `clientMetadataID`:
```

Comment:
It's a bit ambiguous what the client metadata ID is, what merchants should do with the client metadata ID, what the `result` is, and how that relates to the `device data` that our merchants are already familiar with.